next.config.mjs

```
@@ -9,22 +9,37 @@ const nextConfig = {
-      },
-    ];
-  },
+  
+  // For desktop app, we need to use static export
+  output: process.env.TAURI_ENABLED === 'true' ? 'export' : undefined,
+  
+  // Next will use a different output directory for Tauri builds
+  distDir: process.env.TAURI_ENABLED === 'true' ? 'out' : '.next',
+  
+  // Web-only features
+  ...(process.env.TAURI_ENABLED !== 'true' ? {
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)